✨ feat(extension): Add the ability to run an "autoLaunch" prompt as soon as the extension activates

hassoncs opened the pull request:
This adds the ability to automatically start a prompt right when VSCode is launched. When the extension activates, it will check for a file `.kilocode/launchPrompt.md`. If found, it will immediately start executing the prompt.

This allows us to do some really cool testing automations– this would give the extension the ability to test itself! You could write a prompt into a dir, then launch `code` to that dir. The prompt could tell Kilo to test out it's own functionality and then write out a bug report when it's done.

If there's a better way to trigger prompts automatically, please let me know! But this feature would let me continue to experiment in my own dev workflows and eventually Kilo can fully build itself!

- integrate maybeRunAutoLaunchPrompt to auto launch prompt if available
- create autoLaunchPrompt utility to check and execute launchPrompt.md file

hassoncs committed:
✨ feat(extension): Run an "autoLaunch" prompt as soon as the extension activates

This adds the ability to automatically start a prompt right when VSCode is launched. When the extension activates, it will check for a file `.kilocode/launchPrompt.md`. If found, it will immediately start executing the prompt.

This allows us to do some really cool automations– this would give the extension the ability to test itself. You could write a prompt into a dir, then launch `code` to that dir. The prompt could tell Kilo to test out it's own functionality and then write out a bug report when it's done.

If there's a better way to trigger prompts automatically, please let me know! But this feature would let me continue to experiment in my own dev workflows and eventually Kilo can fully build itself!

- integrate maybeRunAutoLaunchPrompt to auto launch prompt if available
- create autoLaunchPrompt utility to check and execute launchPrompt.md file

chrarnoldus commented on src/extension.ts:
```diff
@@ -156,6 +157,8 @@ export async function activate(context: vscode.ExtensionContext) {
 		context.subscriptions.push(watcher)
 	}
 
+	maybeRunAutoLaunchPrompt(context) // kilocode_change
```
Should there be await here?

hassoncs replied:
I didn't want to block the rest of the extension loading, but I agree it's a little confusing because it is an async function. Maybe I could name it better?

Commit:
♻️ refactor(extension): update auto-launch task logic

- rename maybeRunAutoLaunchPrompt to checkAndRunAutoLaunchingTask for clarity
- add await to checkAndRunAutoLaunchingTask
